### PR TITLE
solution isPasswordActual

### PR DESCRIPTION
**Pull request by DemianZolotukhin**

### CODE REVIEW

File: src/isPasswordActual.test.js [hunk 1]
```diff
@@ -24,5 +25,35 @@ describe(`Function 'isPasswordActual':`, () => {
...
+      .toBe('Password is actual.');
+  });
+
+  it(`should return 'Password is actual' 
+  if string used as argument`, () => {
+    const lastYear = isPasswordActual('20g4', '8', '6');
+
+    expect(lastYear)
+      .toBe('Password is actual.');
+  });
```
[the remark this answers is not in the document]
Comment (DemianZolotukhin) [reply src/isPasswordActual.test.js:58]: so I should correct function?

**Review by Ivan-Shvorak — CHANGES_REQUESTED** [review]
Please add more tests using boundary values
